### ISSUE TITLE
loss fluctuates during training

### ISSUE THREAD

**Issue by cailk** [facebookresearch/mae#21]
Thanks for the great work!

I wanted to pre-train a _mae_vit_base_patch16_ model on imagenet-1k, and I found the loss fluctuates during training. I was wondering if this is normal for pre-training mae.
<img width="308" alt="截屏2022-01-20 15 19 52" src="https://user-images.githubusercontent.com/17965136/150294497-1e49bcca-04a0-41f3-b38c-4a226d0e8c96.png">

**Comment by KaimingHe** [comment]
It looks like you zoom in the plot in between 303k and 309k, and zoom in the loss value near 0.42. If you plot the full training, you can see it is overall smooth. It also looks like your tensorboard plots more points after 305k. You may check whether you changed the batch size (so it may plot more frequently), or whether your job is resumed after 305k. In any case, the fluctuation is small and just looks big because it is zoomed in.

The reference curve is as follows:
<img width="1200" alt="Screen Shot 2022-01-20 at 12 05 03 AM" src="https://user-images.githubusercontent.com/11435359/150297834-5fcc9213-bc50-462a-b580-a7b5a7492594.png">

**Comment by cailk** [comment]
> It looks like you zoom in the plot in between 303k and 309k, and zoom in the loss value near 0.42. If you plot the full training, you can see it is overall smooth. It also looks like your tensorboard plots more points after 305k. You may check whether you changed the batch size (so it may plot more frequently), or whether your job is resumed after 305k. In any case, the fluctuation is small and just looks big because it is zoomed in.
> 
> The reference curve is as follows: <img alt="Screen Shot 2022-01-20 at 12 05 03 AM" width="1200" src="https://user-images.githubusercontent.com/11435359/150297834-5fcc9213-bc50-462a-b580-a7b5a7492594.png">

OK! Looks like there is something wrong with my tensorboard. Thanks a lot!

**Comment by klinic** [comment]
Excuse me, How many GPUS did you use for this pre-training?

**Comment by cailk** [comment]
> Excuse me, How many GPUS did you use for this pre-training?

Hi, I just used a single machine with 8 V100 GPUs to train the base model.
